mandiantadvantagethreatintelligence_connector.py

```diff
@@ -214,6 +220,14 @@ def _handle_indicator_lookup(self, param, action_result):
             self.save_progress("Error getting indicator info")
             return action_result.set_status(phantom.APP_ERROR, "Error getting indicator info")
 
+        if not response.get("indicators"):
```

Comment:
Only one condition would suffice for no indicators present:

if not response.get("indicators", []):
    self.save_progress("No indicators retrieved from Mandiant")
    return action_result.set_status(phantom.APP_SUCCESS)